scripts/zones/Horlais_Peak/mobs/Helltail_Harry.lua

```diff
@@ -8,8 +8,8 @@ require("scripts/globals/status")
 local entity = {}
 
 entity.onMobSpawn = function(mob)
-    mob:setMod(xi.mod.SILENCERES, 90)
-    mob:setMod(xi.mod.LULLABYRES, 70)
+    mob:setMod(xi.mod.SILENCE_MEVA, 900)
```

Comment:
Is this base 1000 does anyone know?

Reply:
This is part of their new magic rewrite - we should not include any of this I believe